client/src/components/helpers/loader.css

```
@@ -2,6 +2,7 @@
   height: 100%;
   width: 100%;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
```

`width` and `height` can go, because they are inherited from the body, and `gap` can replace the `br` element.

```suggestion
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  gap: 1em;
```

---

I would avoid changing things that are not a part of this PR.